AlphaWallet/Settings/ViewControllers/SettingsViewController.swift

```diff
@@ -1,7 +1,7 @@
 // Copyright © 2018 Stormbird PTE. LTD.
 
 import UIKit
-import PromiseKit
+import Combine
```

Comment:
There seems to be quite a bit of unrelated changes in the PR. Are all of them needed to fix it?

Can you also rename the PR title and commit? The original issue title doesn't describe well what was the problem.